### ISSUE TITLE
i am not able to run code for installation

### ISSUE THREAD

**Issue by rakshitsharma574** [microsoft/Bringing-Old-Photos-Back-to-Life#267]
git : The term 'git' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:2 char:1
+ git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
+ ~~~
    + CategoryInfo          : ObjectNotFound: (git:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
 
Copy-Item : A parameter cannot be found that matches parameter name 'rf'.
At line:3 char:4
+ cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
+    ~~~
    + CategoryInfo          : InvalidArgument: (:) [Copy-Item], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.CopyItemCommand

**Comment by vionomad** [comment]
Sounds like you are on a Windows system and not Linux. And the 'git' package is not installed.

**Comment by justsomeguy564** [comment]
I am on a Windows system and I have git. I installed using 
git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
but the command
cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
doesn't work for me. cp is a linux command for copy.

Can this be run in Windows? 

If so, it looks like the line is trying to copy the contents of the Synchronized-BatchNorm-PyTorch/sync_batchnorm directory up one level.

Is that correct?

**Comment by Aspudden** [comment]
Just for information, 
I have the same problem as you justsomeguy564, but in my case I try to clone MS code .. 

git clone https://github.com/MicrosoftDocs/mslearn-create-razor-pages-aspnet-core

I'm also on Windows system, have installed also git but still error ..
